src/System.Net.Sockets/src/System/Net/Sockets/Socket.Unix.cs

```diff
@@ -17,5 +21,136 @@ private Socket GetOrCreateAcceptSocket(Socket acceptSocket, bool unused, string
             handle = null;
             return null;
         }
+
+        private void SendFileInternal(string fileName, byte[] preBuffer, byte[] postBuffer, TransmitFileOptions flags)
+        {
+            // Note, UseDefaultWorkerThread is the default and is == 0.
+            // Unfortunately there is no TransmitFileOptions.None.
+            if (flags != TransmitFileOptions.UseDefaultWorkerThread)
+            {
+                throw new PlatformNotSupportedException();
```

Comment:
It'd be good to add a message to this PNSE.  While we've probably strayed from this in some places, we've tried to make it so that if the PNSE is thrown based on some subset of usage (e.g. a particular parameter value isn't supported), then the PNSE has a message explaining that.  If any possible usage of the member throws the PNSE, then we've been ok with not including a custom message.